adobe_vipm/flows/global_customer.py

```diff
@@ -321,22 +321,22 @@ def create_gc_agreement_deployment(
         return agreement_deployment.agreement_id
 
     try:
-        address = adobe_customer["companyProfile"]["address"]
+        address = adobe_customer["companyProfile"].get("address", {})
         contact = adobe_customer["companyProfile"]["contacts"][0]
         param_address = {
-            "country": address["country"],
-            "state": address["region"],
-            "city": address["city"],
-            "addressLine1": address["addressLine1"],
-            "addressLine2": address["addressLine2"],
-            "postCode": address["postalCode"],
+            "country": address.get("country", ""),
+            "state": address.get("region", ""),
+            "city": address.get("city", ""),
+            "addressLine1": address.get("addressLine1", ""),
+            "addressLine2": address.get("addressLine2", ""),
+            "postCode": address.get("postalCode", ""),
```

Comment:
The same comment as in "shared.py". Also, it seems a repeated piece of code maybe it would be useful to create a helper method and reuse?